test/kinetics/kineticsFromYaml.cpp

```diff
@@ -605,8 +605,23 @@ TEST(Kinetics, InterfaceKineticsFromYaml)
     auto& cov_map = coverage_deps["H(s)"].as<Cantera::AnyMap>();
     EXPECT_DOUBLE_EQ(cov_map["E"].asDouble(), -6e6);
 
+
     auto R3 = kin->reaction(2);
     EXPECT_TRUE(std::dynamic_pointer_cast<StickingArrheniusRate>(R3->rate()));
+
+    auto soln2 = newInterface("surface-phases.yaml", "Pt-multi-sites");
+    auto kin2 = soln2->kinetics();
+    auto R4 = kin2->reaction(3);
+    vector_fp coeffs{1.0e6, 3.0e6, -7.0e7, 5.0e6};
+    const auto rate4 = std::dynamic_pointer_cast<InterfaceArrheniusRate>(R4->rate());
+    Cantera::AnyMap coverage_deps2;
+    rate4->getCoverageDependencies(coverage_deps2);
+    coverage_deps2.applyUnits();
+    auto& cov_map2 = coverage_deps2["O(s)"].as<Cantera::AnyMap>();
```

Comment:
The `Cantera::` prefix is unnecessary here since we have a `using namespace Cantera` directive at the top of the file.